### PR TITLE
Dev sir 1017 grpc streams events

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Api.Common/Configuration/TenantRateLimitOptions.cs [hunk 1]
```diff
@@ -9,5 +9,7 @@ public TenantRateLimitOptions()
         }
 
         public string ClientPolicyPrefix { get; set; }
+
+        public int StreamsPerTenant { get; set; }
```
Comment (KonstantinRyazantsev) [inline src/Api.Common/Configuration/TenantRateLimitOptions.cs:13]: I think just `Streams` is enough here. The containing class says to us already, that it's about tenants rate limiting.

File: src/Api.Worker/Api.Worker.csproj [hunk 2]
```diff
@@ -22,7 +22,7 @@
     <PackageReference Include="MassTransit.Extensions.DependencyInjection" Version="6.3.2" />
     <PackageReference Include="MassTransit.RabbitMQ" Version="6.3.2" />
     <PackageReference Include="swisschain.extensions.Postgres" Version="1.0.0" />
-    <PackageReference Include="Swisschain.Sdk.Server" Version="6.7.0" />
+    <PackageReference Include="Swisschain.Sdk.Server" Version="6.8.0-beta.2" />
```
Comment (KonstantinRyazantsev) [inline src/Api.Worker/Api.Worker.csproj:25]: Relesase it

File: src/Api/GrpcServices/Streams/DepositUpdatesStreamService.cs [hunk 3]
```diff
@@ -7,9 +10,31 @@ namespace Api.GrpcServices.Streams
...
+        public DepositUpdatesStreamService(RateLimiterForStreams rateLimiterForStreams,
+            ILogger<DepositUpdatesStreamService> logger, bool needPing = true) 
             : base(logger, needPing)
         {
+            _rateLimiterForStreams = rateLimiterForStreams;
+        }
+
+        protected override Task BeforeStreamRegistered(StreamInfo<DepositUpdateArrayResponse> streamInfo)
+        {
+            var tenantId = streamInfo.Keys.FirstOrDefault();
```
Comment (KonstantinRyazantsev) [inline src/Api/GrpcServices/Streams/DepositUpdatesStreamService.cs:24]: is it legit to have null in tenant ID, it looks like it should be not null according to its usage

Comment (OttoVT) [reply src/Api/GrpcServices/Streams/DepositUpdatesStreamService.cs:24]: it is not legit to have null

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**